python/Gaffer/__init__.py

```diff
@@ -61,3 +61,6 @@
 
 # Class-level non-UI metadata registration
 Metadata.registerValue( Reference, "childNodesAreReadOnly", True )
+
+# Patch to allow loading of gaffer scenes with Animation.Type renamed to Animation.Interpolation
+Animation.Interpolation = Animation.Type
```

Comment:
We put this sort of patch in a separate config file to keep it separate from the official API (for example https://github.com/GafferHQ/gaffer/blob/master/startup/Gaffer/switchCompatibility.py). It lets us quickly find all the monkey patches and periodically retire them from service. Let's move this one to a new `startup/Gaffer/animationCompatibility.py` file.